### PR TITLE
Introducing initial helm-chart for version 0.21.5

### PR DESCRIPTION
**Pull request by abhishek44sharma**

### CODE REVIEW

**Review by v0lkan — APPROVED** [review]
Looking good, as far as yamls go :).

File: helm-charts/0.21.5/values.yaml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+podAnnotations:
+  "helm.sh/hook": pre-install
+
+replicaCount: 1
+
+image:
+  repository: nginx
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```
Comment (v0lkan) [inline helm-charts/0.21.5/values.yaml:69]: do we need these?
and why NGINX :) ?

File: helm-charts/0.21.5/values.yaml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
```
Comment (v0lkan) [inline helm-charts/0.21.5/values.yaml:84]: `podAnnotations` is defined for the second time.

File: helm-charts/0.21.5/values.yaml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
```
Comment (v0lkan) [inline helm-charts/0.21.5/values.yaml:101]: I’d recommend cleaning the attributes that we don’t use for simplicity and clarity.

**Review by v0lkan — CHANGES_REQUESTED** [review]
had some change requests and questions.

File: helm-charts/0.21.5/values.yaml [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+  baseImage: distroless
+  registry: vsecm
+  logLevel: "7"
+  images:
+    safe:
+      distrolessRepository: vsecm-ist-safe
+      distrolessFipsRepository: vsecm-ist-fips-safe
+      photonRepository: vsecm-photon-safe
+      photonFipsRepository: vsecm-photon-fips-safe
+      tag: 0.21.4
```
[the remark this answers is not in the document]
Comment (abhishek44sharma) [reply helm-charts/0.21.5/values.yaml:23]: Yes @v0lkan, this is initialisation MR. We'll make updates to this cart.

File: helm-charts/0.21.5/values.yaml [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+    sentinel:
+      distrolessRepository: vsecm-ist-sentinel
+      distrolessFipsRepository: vsecm-ist-fips-sentinel
+      photonRepository: vsecm-photon-sentinel
+      photonFipsRepository: vsecm-photon-fips-sentinel
+      tag: 0.21.4
+      pullPolicy: IfNotPresent
+    spireAgent:
+      repository: ghcr.io/spiffe/spire-agent
+      tag: 1.8.0
```
[the remark this answers is not in the document]
Comment (abhishek44sharma) [reply helm-charts/0.21.5/values.yaml:34]: Yes @v0lkan will submit MR with those updates. This is just initialisation auto generated MR.

File: helm-charts/0.21.5/values.yaml (hunk 1 above)
Comment (abhishek44sharma) [reply helm-charts/0.21.5/values.yaml:69]: We can get rid of it, will update.

File: helm-charts/0.21.5/values.yaml (hunk 2 above)
Comment (abhishek44sharma) [reply helm-charts/0.21.5/values.yaml:84]: will remove.

File: helm-charts/0.21.5/values.yaml (hunk 3 above)
Comment (abhishek44sharma) [reply helm-charts/0.21.5/values.yaml:101]: +1, will do. Thanks!! 👍

**Review by v0lkan — APPROVED** [review]
🌮 .

**Pull request merged**